test/integration/addons_test.go

```diff
@@ -267,9 +267,17 @@ func validateHelmTillerAddon(ctx context.Context, t *testing.T, profile string)
 		t.Fatalf("wait: %v", err)
 	}
 
+	if NoneDriver() {
+		_, err := exec.LookPath("socat")
```

Comment:
How about put a nice warning message for users too ? That they need this ?

Reply:
Honestly, it seems like a pretty invasive peek into implementation details for kubectl. I can't reliably determine which versions of kubectl will need socat on which platforms.

Reply:
or how about installing it on our integraiton machines?